bin/node/runtime/src/lib.rs

```diff
@@ -175,7 +175,7 @@ impl pallet_utility::Trait for Runtime {
 }
 
 parameter_types! {
-	pub const MaximumWeight: Weight = 2_000_000;
+	pub const MaximumWeight: Weight = Perbill::from_percent(80) * MaximumBlockWeight::get();
```

Comment:
```suggestion
	pub const MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * MaximumBlockWeight::get();
```

Reply:
Looking at this isolated (in unified diff view) raised the question: "How is this different from the `MaximumBlockWeight`".